src/ucp/wireup/select.c

```diff
@@ -1469,29 +1469,38 @@ ucs_status_t ucp_wireup_select_sockaddr_transport(ucp_ep_h ep,
     ucp_rsc_index_t tl_id;
     ucp_md_index_t md_index;
     uct_md_h md;
+    int i;
+
+    /* Go over the sockaddr transports priority list and try to use the transports
+     * one by one for the client side */
+    for (i = 0; i < context->config.num_sockaddr_tls; i++) {
+        for (tl_id = 0; tl_id < context->num_tls; ++tl_id) {
```

Comment:
need to use `ucs_for_each_bit(tl_id, context->tl_bitmap) {` here
(it is a bug in the old code)

Reply:
@brminich good catch, 10x!